### ISSUE TITLE
Project draft status

### ISSUE THREAD

**Issue by ewels** [ScilifelabDataCentre/dds_web#509]

**Comment by ewels** [comment]
We need to be able to set different statuses for projects. The suggestions that pop to mind are:

* `draft` - default for new projects when created by facilities. User is not notified and does not have access. Used whilst facilities upload data and prepare project.
* `published` - set when the project is ready for access. When set, user is notified and gets access.
* `closed` - automatically set when project reaches end of life. User is warned then notified, loses access. Facility maintains access. Used for a grace period prior to deletion.

**Comment by inaod568** [comment]
#484 is a subtask to this

**Comment by ewels** [comment]
Sounds like a duplicate to be honest? ie. basically the same thing.

**Comment by inaod568** [comment]
Yes, but I added it a while back just as a minor task to add the fields to the db and the web interface. To me this card is more the process of actually adding the changing functionality.

**Comment by ewels** [comment]
I'm not totally up to speed with the whole task / subtask etc structure so whatever makes most sense for you 👍🏻
